docs/community/100-days-of-oss.md

```diff
@@ -88,23 +109,29 @@ Our hope is that the community will provide a safe space to discuss challenges,
 5. **Making a Positive Impact**: Your contributions benefit other contributors who rely on these projects, fostering a sense of fulfillment and giving back to the community.
 
 ## Events
```

Comment:
Perhaps we want to consider to have an independent section in the `community` as a one-stop place for all of our events rather than put them as a `100DaysOfOSS` section.
That way, folks can be aware of the section and directly check our ongoing events.

Reply:
We added the calendar link in community. That should be good for now.